recipes/python-switch-quotes

```diff
@@ -0,0 +1,4 @@
+(python-switch-quotes :fetcher github
+                      :repo "werehuman/python-switch-quotes"
+                      :files ("python-switch-quotes.el"
```

Comment:
```lisp
("*.el" "*.el.in" "dir"
 "*.info" "*.texi" "*.texinfo"
 "doc/dir" "doc/*.info" "doc/*.texi" "doc/*.texinfo"
 (:exclude ".dir-locals.el" "test.el" "tests.el" "*-test.el" "*-tests.el"))
```

Could you rename or move test file name and remove `:files` attribute ? Current test file name is not matched against default `:exclude` pattern(above snippet).
- Renaming: `test-python-switch-quotes.el` -> `python-switch-quotes-test.el`
- Moving: `test-python-switch-quotes.el` -> `test/test-python-switch-quotes.el`

And I suppose README.md is not necessary. It is enough by package comment and docstring.